apps/epic-web/src/components/app/layout.tsx

```diff
@@ -68,6 +71,7 @@ const Layout: FunctionComponent<React.PropsWithChildren<LayoutProps>> = ({
         noindex={noIndex}
       />
       <GlobalSearchBar />
+      <RedeemDialogForCoupon />
```

Comment:
no need to validate/conditional since it is done behind the scenes already

uses layout, so any page using layout "just works"